specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -2425,6 +2494,18 @@
       "description": "AAD (Azure Active Directory) data connector properties.",
       "type": "object"
     },
+    "AADCheckRequirements": {
```

Comment:
>AADCheckRequirements [](start = 5, length = 20)

not used anywhere.. remove ?

Reply:
Is this used in some other file ?.. same with other definitions added with this change

---
In reply to: [363532156](https://github.com/Azure/azure-rest-api-specs/pull/8020#discussion_r363532156) [](ancestors = 363532156)

Reply:
Looks like this need to use discriminator pattern as others, based on kind...  for ex: this one has additional property (tenantId), compared to others that inherited from DataConnectorsCheckRequirements

---
In reply to: [363532368](https://github.com/Azure/azure-rest-api-specs/pull/8020#discussion_r363532368) [](ancestors = 363532368,363532156)